js/interface.js

```diff
@@ -121,7 +121,9 @@ Fliplet().then(function () {
     ].join(' '),
     setup: function (ed) {
       ed.on('keyup paste', function(e) {
-        data.html = ed.getContent();
+        data.html = ed.getContent()
+          .replace(/&lt;/g, '<')
+          .replace(/&gt;/g, '>');
```

Comment:
Instead of just dealing with `<` and `>`, do we want to just run a **HTML entity decode**?